docs/frontend-system/architecture/00-index.md

```diff
@@ -6,46 +6,74 @@ sidebar_label: Overview
 description: The structure and architecture of the new Frontend System
 ---
 
-> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
+> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins. If you want to use the
```

Comment:
All of these notes, can we convert them to `:::info` admonitions? I think that make them more pronounced.

Reply:
Agreed